catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -59,8 +64,6 @@
  */
 public class FileSystemStorageProvider implements StorageProvider {
```

Comment:
Since the create() and update() methods are nearly identical, I would suggest refactoring them like this: <script src="https://gist.github.com/dcruver/c8d5df8e728a43a3bb350d766dd873b6.js"></script>